charts/timed/values.yaml

```diff
@@ -12,7 +12,7 @@ backend:
     # -- Backend image name
     repository: ghcr.io/adfinis/timed-backend
     # -- Backend version.
-    tag: v3.0.12
+    tag: 3.0.15
```

Comment:
Is the removal of the `v` intended here?

Reply:
yes, since the switch to semrel container images aren't tagged with a v anymore/